app/experimenter/experiments/models.py

```diff
@@ -512,6 +513,13 @@ def completed_addon(self):
     def completed_variants(self):
         return self.variants.exists()
 
+    @property
+    def completed_branched_addon(self):
```

Comment:
1) We should also `is_branched_addon`
2) `all([v.addon_release_url for v in self.variants.all()])`